packages/ColorPicker/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@igloo-ui/color-picker",
+  "version": "0.0.0",
+  "main": "dist/ColorPicker.js",
+  "module": "dist/ColorPicker.js",
+  "types": "dist/ColorPicker.d.ts",
+  "styles": "dist/color-picker.css",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "git+https://github.com/gsoft-inc/ov-igloo-ui.git",
+  "author": "Officevibe Inc.",
+  "contributors": [
+    "Vicky Comeau vicky.comeau@gsoft.com"
+  ],
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": ">= 16.8.6"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup -c rollup.config.js"
+  },
+  "dependencies": {
+    "@igloo-ui/dropdown": "^1.6.1",
+    "@igloo-ui/tokens": "^2.0.0",
```

Comment:
You often use `@igloo-ui/color` in the project, I find it strange to don't see it in the dependencies

Reply:
this is a mistake, good catch!

Reply:
It might have been cached?